cli/ingress-controller/flags.go

```diff
@@ -158,6 +159,8 @@ Kong's Admin SSL certificate.`)
 	// Resource filtering
 	flags.String("watch-namespace", apiv1.NamespaceAll,
 		`Namespace to watch for Ingress. Default is to watch all namespaces`)
+	flags.Bool("skip-empty-annotation", false,
```

Comment:
Please rename this flag to `skip-classless-ingress-v1beta1`.
We plan to introduce another flag to control this behavior for Ingress v1 as well.